### DISCUSSION TITLE
Detecting Minor Changes with `cargo-semver-checks`

### DISCUSSION THREAD

**Discussion by wmmc88** [release-plz/release-plz#1883] [Q&A] [answered]
It looks like `release-plz` classifies the output of `cargo-semver-checks` t[o either result in no bump, or a major bump](https://github.com/release-plz/release-plz/blob/7f8de507c1bc35033a661329234ceb8cb856ca19/crates/release_plz_core/src/semver_check.rs#L94). Why doesn't it consider output that suggests minor or patch bumps?

[This](https://github.com/release-plz/release-plz/blob/7f8de507c1bc35033a661329234ceb8cb856ca19/README.md?plain=1#L14) seems to imply that the new version should be calculated from both the output of `cargo-semver-checks` and the conventional commits

**Comment by marcoieni** [comment, marked as answer]
Patch bumps are the default. So it doesn't matter.

Pr welcome to handle minor changes 👍
